### DISCUSSION TITLE
Please help me to recover my account

### DISCUSSION THREAD

**Discussion by RodrigoLorca** [community/community#119713] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My Microsoft authenticator isn't working properly. I'm logged within the app, but the codes aren't working so I cannot login in my personal GitHub account. Can anyone help me? Please, just imagine for a moment losing all your work

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @RodrigoLorca ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by Bonavigoo** [comment]
The same happened to me.
I've bought a new phone, because the last one broke, and restored Authenticator.
My GitHub in my browser signed out, and now I only have GitHub Desktop logged in, and that doesn't help much... I've lost the .txt file with the key.
Could you find any solutions?
